bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -133,25 +133,31 @@ pub mod impls {
 		}
 	}
 
-	pub struct Author;
-	impl OnUnbalanced<NegativeImbalance> for Author {
-		fn on_nonzero_unbalanced(amount: NegativeImbalance) {
+	pub struct ToAuthor;
+	impl OnUnbalanced<RingNegativeImbalance> for ToAuthor {
+		fn on_nonzero_unbalanced(amount: RingNegativeImbalance) {
+			let numeric_amount = amount.peek();
+			let author = Authorship::author();
 			Ring::resolve_creating(&Authorship::author(), amount);
```

Comment:
Can we replace `Authorship::author()`  with `author` last line?

Reply:
Just keep this.

Easy review while compare to Polkadot.